tests/sentry/api/endpoints/test_organization_metric_details.py

```diff
@@ -161,15 +165,25 @@ def test_same_entity_multiple_metric_ids(self):
                 errors=2,
             )
         )
+        response = self.get_response(
+            self.organization.slug,
+            "derived_metric.multiple_metrics",
+        )
+        assert response.status_code == 404
+        assert response.json()["detail"] == (
+            "Not all the requested metrics or the constituent metrics in "
+            "['derived_metric.multiple_metrics'] have data in the dataset"
+        )
+
         self._send_buckets(
             [
                 {
                     "org_id": self.organization.id,
                     "project_id": self.project.id,
-                    "metric_id": resolve_weak("metric1"),
-                    "timestamp": time.time(),
+                    "metric_id": indexer.record("metric_foo_doe"),
+                    "timestamp": int(time.time()),
                     "tags": {
-                        indexer.record("release"): indexer.record("foo"),
+                        resolve_weak("release"): indexer.record("foo"),
```

Comment:
Why not use `indexer.record` here? Are we sure that the string exists?

Reply:
Yeah because store_session on 159 creates it, no?

Reply:
Right.